encryptor/encryptors/vigenere.py

```diff
@@ -1,48 +1,90 @@
-from ..errors.not_allowed_value import NotAllowedValue
+"""
+Encryptor - Vigenere
+
+https://en.wikipedia.org/wiki/Vigen%C3%A8re_cipher
+"""
+
 from itertools import cycle
 
+from ..errors.not_allowed_value import NotAllowedValue
+
 
 class Vigenere:
-    # square or Vigenère table, also known as the tabula recta, can be used for encryption and decryption.
-    # may be changed for more secure
+
+    """
+    square or Vigenère table, also known as the tabula recta,
+    can be used for encryption and decryption.
+    may be changed for more secure
+    """
+
     # use ASCII by default
     ALPHABET = {i: chr(i) for i in range(128)}
-    KEY = 'abra'
+    KEY = "abra"
 
     @staticmethod
     def encrypt(text: str, key: str = KEY, tabula_recta=None):
+        """
+        Viegenere - Encrypt
+
+        Call helper functions to encrypt input string
+        """
+
         if tabula_recta is None:
             tabula_recta = Vigenere.ALPHABET
         if key.isalpha():
```

Comment:
We could use some spacing here
```suggestion
            tabula_recta = Vigenere.ALPHABET
        
        if key.isalpha():
```